tasks.php

```diff
@@ -582,7 +583,7 @@ function create_task_from_form_submission($formsub)
                 // If we successfully create the task, we should reload
                 //   the page to clear the POST data and make sure that
                 //   reloading does not lead to duplicated tasks.
-                header("Location: $code_url/tasks.php");
+                metarefresh(0, "$code_url/tasks.php");
```

Comment:
There already exists a variable $tasks_url which you could use instead of $code_url/tasks.php.

Reply:
Thanks! Fixed in dfa3273172502edbe898e5c978c01025f7501666 and f49ee4e079d3571be2a841eee4cb9061b4c3a8be :)